src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './SearchForm.module.scss';
+import { Button } from '../Button';
+
+export const SearchForm = () => {
+  const searchHandle = () => {
+    console.log('searchHandle');
+  };
+
+  return (
+    <>
+      <h1>Find your movie</h1>
+      <div className={styles.searchForm}>
+        <input
+          placeholder="What do you want to watch?"
+          className={styles.searchFormInput}
+          type="text"
+        />
```

Comment:
input witout onChange?

Reply:
we decided to leave it for future functionality. From now on , we will not create functions in advance until we need them .Will be added when functionality will be implemented